resources/ts/Components/CopyText.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ComponentProps, useRef, useState } from "react";
+import { Tooltip } from "./Tooltip";
+import { ButtonIconCopy } from "./ButtonIconCopy";
+import { useTranslation } from "react-i18next";
+
+export type Props = ComponentProps<"div">;
+
+export const CopyText = ({ children }: Props) => {
```

Comment:
If it is only children where using as prop, you can better define the type like so:
```ts
type Props = { children: ReactNode }
```